### ISSUE TITLE
Reverse Engineering - HasDefaultValueSql is not generated anymore

### ISSUE THREAD

**Issue by StefanFalz** [ErikEJ/EFCorePowerTools#513]
Prior versions of EFCore Power Tools generated a call to HasDefaultValueSql("default value") properties with a configured default value in the database.

I use .NET Core 3.1 (SDK 3.1.402 is installed).

Please add the HasDefaultValueSql method again. Perhaps as an optional setting.

### Steps to reproduce
Create a Microsoft SQL Server database with a table "Test", add a column "DateCreated", edit the column properties and set a default value of GETDATE() for this column. Reverse engineer this database.

Prior versions (unfortunately I don't know which version exactly because I didn't use the reverse engineering for a few months as there were no database changes) generating the following code:

```csharp
modelBuilder.Entity<Test>(entity =>
{
    ...

    entity.Property(e => e.DateCreated).HasDefaultValueSql("(getdate())");
});
```

The current version produces:

```csharp
modelBuilder.Entity<Test>(entity =>
{
    ...
});
```

### Further technical details
EF Core Power Tools version: 2.4.217.0 (none of the preview options are checked in the settings dialog)

Database engine: SQL Server 2017

Visual Studio version: VS 2019, 16.7.4

**Comment by StefanFalz** [comment]
Your extension is really awesome and I use it for years now.

But in my case this is a serious bug because each of my 200+ tables contains at least one column with a default value in SQL Server. After updating the model and the context classes there is no way to create a new record unless I'll set the DateCreated property manually.

Please fix this bug.

**Comment by ErikEJ** [comment]
Please provide a CREATE TABLE script, and I will have a look.

**Comment by StefanFalz** [comment]
> 
> 
> Please provide a CREATE TABLE script, and I will have a look.

I'll try to create a repro project with a test table. I'll be back in a few minutes.

**Comment by StefanFalz** [comment]
I think I've figured it out.

The SQL Server user used for reverse engineering is db_datareader and db_datawriter in this specific database. The reverse engineering doesn't create HasDefaultValueSql calls.

All other information (comments, indexes, keys, ...) are generated. Only the default values are missing.

I've created a test database and in this case the same user is db_owner. Now the reverse engineering creates HasDefaultValueSql calls.

So my (new) question is: Is it neccessary to use a "db_owner" user for reverse engineering?
